product_sequence/models/ir_sequence.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class IrSequence(models.Model):
+    _inherit = "ir.sequence"
+
+    @api.model
+    def get_category_sequence_id(self, category=False):
+        if self.env.user.company_id.use_parent_categories_to_determine_prefix:
+            while not category.sequence_id and category.parent_id:
```

Comment:
This code is not robust as category can be False (default value). You should add the condition "while category and ..." OR you should remove the default value of category to make sure that this parameter is set.

Reply:
Done